lib/modulr/resources/payments/payments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Modulr
+  module Resources
+    module Payments
+      class Payments < Collection
```

Comment:
Here I want to confirm the logic we have started applying in the core. This should be 

`Modulr::Resources::Payments::Collection` (and subclass `BaseCollection`)

Are we ok here? @devengoapp/dev